### DISCUSSION TITLE
Parent Grain reconstruction

### DISCUSSION THREAD

**Discussion by huseyinhzl** [mtex-toolbox/mtex#1687] [Ask Anything]
Hello all,

I am trying to do parent grain reconstruction in Ti6Al4V alloy and my modified Ti6Al4V. I am currently using ORTools 2.0.2 and MTEX 5.8.2 for this process, yet I have problems during reconstruction which I do not know the reason for.  The code looks working fine for the original Ti6Al4V, but when I add a third phase the reconstruction is not working correctly such that I am getting something like the below. Also, I am getting a weird plot after calculating parent from vote. Some data points are missing like in the second image (I am getting big white areas). So I am wondering if any one can help me how to solve this problems?

![untitled1](https://user-images.githubusercontent.com/120036622/207071377-add3fcb5-546f-446e-9477-17ccaf22ff71.png)
![untitled2](https://user-images.githubusercontent.com/120036622/207075371-5a81a4f0-d875-45ca-8254-bc128f61ef9c.png)

Thank you so much

**Comment by Tijmenvermeij** [comment]
Hi, 
how do you handle the third phase? Do you leave it in the EBSD and GRAINS variable, before creating the `parentGrainReconstructor`? 
Also, does the third phase have a clearly different `crystalSymmetry`?

You may need to post your code and data so that someone can check it out...
